test/Makefile

```diff
@@ -35,6 +35,17 @@ build-base-branch:
 	mv $(TMP_REPO)/_output/rpmbuild $(REPO)/_output/rpmbuild-base
 	rm -rf "$(TMP_REPO)"
 
+
+.PHONY: build-previous-minor-version
+build-previous-minor-version: YMINUS1_MINOR =release-$(MAJOR).$(shell expr $(MINOR) - 1)
+build-previous-minor-version: TMP_REPO !=mktemp -d
+build-previous-minor-version:
+	if ! [ -d "$(REPO)/_output" ]; then mkdir $(REPO)/_output; fi
+	git clone --single-branch --branch $(YMINUS1_MINOR) https://github.com/openshift/microshift.git $(TMP_REPO)
```

Comment:
Why do we build previous minor from code? So far we were using already built and released 4.13 RPMs - I think we should continue on doing that

Reply:
Good point. I'll TAL at that approach and work it in here